src/puppetlabs/dujour/version_check.clj

```diff
@@ -88,10 +88,15 @@
                          (merge version-data)
                          json/generate-string)
         url update-server-url
-        {:keys [status body] :as resp} (client/post url
-                                         {:headers {"Accept" "application/json"}
-                                          :body request-body
-                                          :as :text})]
+        {:keys [status body] :as resp} (try
+                                         (client/post url
+                                                     {:headers {"Accept" "application/json"}
+                                                      :body request-body
+                                                      :as :text})
+                                         (catch java.io.IOException e
```

Comment:
You probably want to add an `import` for this exception, and avoid using the fully qualified name here.